src/fs/fs.c

```diff
@@ -40,6 +40,76 @@ LIST_HEAD(list, cfs_fs_node) fs_list = LIST_HEAD_INITIALIZER();
 /* global endpoint operations for cortxfs*/
 static const struct cfs_endpoint_ops *g_e_ops;
 
+int fs_node_init(struct cfs_fs_node *fs_node, struct namespace *ns,
```

Comment:
We can make this API as internal to fs.c ?
`static int fs_node_init(struct cfs_fs_node *fs_node, struct namespace *ns, size_t ns_size)`

Reply:
Done.